### ISSUE TITLE
Error with dropdown layout fail when <!DOCTYPE html>

### ISSUE THREAD

**Issue by raanand-home** [isteven/angular-multi-select#319]
For some reason the layout failed when <!doctype html> is in the header of the file

Ps I am using Chrome( 43.0.2357.134)

# example

<!DOCTYPE html>
<html >
    <head>
        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.11/angular.min.js"></script>
		<link rel="stylesheet" href="https://raw.githubusercontent.com/isteven/angular-multi-select/master/isteven-multi-select.css">
        <script src="http://isteven.github.io/angular-multi-select/js/libs/isteven-multi-select.js"></script>
    </head>
	
	<script>
		var myApp = angular.module( "myApp", [ "isteven-multi-select" ]);
		myApp.controller('demoAngularCtrl', function($scope, $http) {
		$scope.modernBrowsers = [
    {   name: "Opera",              maker: "(Opera Software)",        ticked: false  },
    {   name: "Internet Explorer",  maker: "(Microsoft)",             ticked: false },
    {   name: "Firefox",            maker: "(Mozilla Foundation)",    ticked: false  },
    {   name: "Safari",             maker: "(Apple)",                 ticked: false },
    {   name: "Chrome",             maker: "(Google)",                ticked: false  }
		]; 
		$scope.outputBrowsers = []
	});
		</script>
	<body ng-app="myApp">

		<div ng-controller='demoAngularCtrl'>
		
			<div     
			isteven-multi-select
			input-model="modernBrowsers"
			output-model="outputBrowsers"
			button-label="name"
			item-label="name maker"
			tick-property="ticked">	</div>
		</div>
		</div>
	</body>
</html>

**Comment by isteven** [comment]
Hi,

The manual website uses <!DOCTYPE html>, and it looks OK.
